internal/rootcoord/meta_table.go

```diff
@@ -1381,6 +1386,10 @@ func (mt *MetaTable) OperatePrivilege(tenant string, entity *milvuspb.GrantEntit
 	if entity.Grantor.Privilege == nil || funcutil.IsEmptyString(entity.Grantor.Privilege.Name) {
 		return fmt.Errorf("the privilege name in the grant entity is empty")
 	}
+	conflict, err := mt.CheckWithPrivilegeGroupNames(ctx, entity.Grantor.Privilege.Name)
+	if err != nil || conflict {
+		return fmt.Errorf("the privilege name in the grant entity is conflict with any of the privilege group names")
+	}
```

Comment:
if user try to grant a PrivilegeGroup, this logic will return error?

Reply:
fixed